pyphare/pyphare/pharesee/tovtk.py

```
@@ -0,0 +1,240 @@
+                    )
+                    first = False
+
+                else:
+                    # dataset already created with shape (current_size,3)
+                    # we add b.shape[0] points (=npx*npy) to the first dim
+                    # hence need to resize the dataset.
+                    pointData_b.resize(current_size + data.shape[0], axis=0)
+                    pointData_b[current_size:, :] = data
+                # pass
+
+                current_size += b.shape[0]
```

Ruff (0.8.2)

Undefined name `b`

(F821)